README.md

```diff
@@ -30,14 +30,36 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The main difference is that map returns a new array of elements while passing each element back to the callback.
+the forEach method executes a function once for each array element.
+Map creates a new array with the results of calling a function on every element in the array.
+
 2. What is the difference between a function and a method?
 
+Both are functions.  The difference is that a method is a function inside an object or class.  A function becomes a method when it is inside an object or class.
```

Comment:
not 'inside' a 'class' but more a function that is attached / applied to an Object / String / Array

Reply:
Yes, I used the wrong word 'inside'.  Like you said,  It's more of an attachment applied to an object / string  / array